src/main.rs

```diff
@@ -35,7 +35,11 @@ fn main() {
     match cli::execute() {
         Ok(()) => std::process::exit(0),
         Err(error) => {
-            logging::ensure_log_error(&error);
+            if log::log_enabled!(log::Level::Error) {
+                log::error!("{:?}", error);
+            } else {
+                eprintln!("{:?}", error);
+            }
```

Comment:
Any particular reason this was inlined? The doc string on the method was quite useful. Also, please keep the `LogError` wrapper to print causes in the format `LogError` does.

Reply:
Its a bit weird and one thing I kind of dislike about anyhow:

`{:?}` prints the error with all its causes and a backtrace if available, as explained here:

https://docs.rs/anyhow/1.0.32/anyhow/struct.Error.html#display-representations

And the function was inlined because it was the only usage. I can separate it out again though.

Reply:
I see. I'm curious how this will work when we have both `std::Error` and `anyhow::Error`, although at this level it's only going to be `anyhow`.